ztp/source-crs/validatorCRs/informDuValidatorMaster.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: master
```

Comment:
If the reference configuration uses "worker" for mcp, do we need the master version?